app/src/main/kotlin/com/wire/android/ui/home/conversationslist/ConversationListState.kt

```diff
@@ -12,9 +12,9 @@ data class ConversationListState(
     val callHistory: List<ConversationItem> = emptyList(),
     val unreadMentions: List<ConversationItem> = emptyList(),
     val allMentions: List<ConversationItem> = emptyList(),
-    val newActivityCount: Int = 0,
-    val missedCallsCount: Int = 0,
-    val unreadMentionsCount: Int = 0,
+    val newActivityCount: Long = 0,
+    val missedCallsCount: Long = 0,
+    val unreadMentionsCount: Long = 0,
```

Comment:
Why `Long` and not `Int` ?

Reply:
SQL query is returning Long for count() I did not want to do uncessary mapping to Int I can change it but I do not think it is really necessary